.buildkite/commands/run-ui-tests.sh

```diff
@@ -24,6 +24,26 @@ install_gems
 echo "--- :cocoapods: Setting up Pods"
 install_cocoapods
 
+echo "--- :test-analytics: Configure Test Analytics"
+# Collect data separately for iPhone and iPad
+if [[ $DEVICE =~ ^iPhone ]]; then
+  export BUILDKITE_ANALYTICS_TOKEN=$BUILDKITE_ANALYTICS_TOKEN_UI_TESTS_IPHONE
+else
+  export BUILDKITE_ANALYTICS_TOKEN=$BUILDKITE_ANALYTICS_TOKEN_UI_TESTS_IPAD
+fi
+
+# Temporary notice about UI tests analytics
+#
+# First, remove any previous notice to avoid duplication
+context='ctx-ui-tests-notice'
+buildkite-agent annotation remove --context "$context" \
+  || true # `remove` will 401 if there's no annotation, but we don't want to fail because of it
+# Then, print a fresh notice
+buildkite-agent annotate \
+  'Test Analytics for UI tests are currently unavailable' \
+  --style 'info' \
+  --context "$context"
+
```

Comment:
`buildkite-agent annotate` will [already replace/update any existing annotation with the same context](https://buildkite.com/docs/agent/v3/cli-annotate#:~:text=You%20can%20update%20an%20existing%20annotation%27s%20body%20by%20running%20the%20annotate%20command%20again%20and%20provide%20the%20same%20context%20as%20the%20one%20you%20want%20to%20update.%20Or%20if%20you%20leave%20context%20blank%2C%20it%20will%20use%20the%20default%20context.) (except if you use `--append` explicitly). That's the whole point of providing a `--context`, after all, so you can reference/update annotations later.

The only case where removing an existing annotation would be useful is if you don't plan to replace it with a new one. But since here you update the `$context` annotation unconditionally, there's no point on removing it first.

```suggestion
buildkite-agent annotate \
  'Test Analytics for UI tests are currently unavailable' \
  --style 'info' \
  --context "$context"

```

Reply:
Thank you for the info.

I implemented the suggestion in [`589776d`](https://github.com/woocommerce/woocommerce-ios/pull/7163/commits/589776dd429a04ec0be37e4505e8a381b2f5a0ed)